### ISSUE TITLE
Blog post idea: Product Design? In Government? What? How?

### ISSUE THREAD

**Issue by quarterback** [18F/product-design#20]
wRITE MORE RON

**Comment by nickttng** [comment]
YES WRITE MORE RON AND MAYHAPS SNEAK IN SOME OF THAT BREATH OF MUSICAL KNOWLEDGE OF YOURS

**Comment by nickttng** [comment]
I think this could become a series exploring from multiple angles like these:
- Within 18F/TTS
  - What and how within 18F design chapter
  - What and how within 18/TTS engineering, product, etc 
- Within the government, e.g. federal, state, etc.
  - What and how with the current and past agency partners
  - What and how other product designers (if they exist) practice their discipline at other agencies
  - (If appropriate, learn from other gov't vendors)
- In what ways does product design in 18F differ and/or share similar traits with product design in private sector?
 
Sounds like a long term project/documentation, so maybe in our 1:1s, we can carve out a time to document/discuss and continue async if it helps.
